fac/pkgsrc-fa
# Node sass bindings

** This is built against node 6 and will need to be built against node 8 when that comes into usage here. **

To test, install package then:

```
[root@pkg-test ~]# pkgin remove nodejs
[root@pkg-test ~]# pkgin -y install nodejs-6.11.0
[root@pkg-test ~]# SASS_BINARY_PATH=/opt/local/share/fa-node-sass/binding.node npm install node-sass
[...]
[root@pkg-test ~]# SASS_BINARY_PATH=/opt/local/share/fa-node-sass/binding.node ./node_modules/.bin/node-sass --version
node-sass       4.5.3   (Wrapper)       [JavaScript]
libsass         3.5.0.beta.2    (Sass Compiler) [C/C++]
```

See also https://github.com/fac/freeagent/pull/12102/files#diff-6dc906be0643182e5bae6aa1506221e6R33 (webpacker upgrade with config to set SASS_BINARY_PATH)
